tables/intro-table/step3.md

```diff
@@ -8,21 +8,48 @@ Now that you have the basic table structure and imports in place, let's express
 
 For example: `"First column"` becomes `{ title: "First column" }`
 
-2) Now apply this same type of transformation to each of your row definitions.
+Your columns definition should now look like:
 
-You're ultimately replacing the string array with an array of objects, where the cell property of the object is the row array you had previously.
+<pre class="file">
+const columns = [
+  { title: "First column" },
+  { title: "Second column" },
+  { title: "Third column" }
+];
+</pre>
+
+Next we'll apply a similar type of transformation to convert each of your row definitions into object representations.
+
+3) Locate the following code just below your columns definition `const defaultRows = [`. Notice that each row in your table is defined by a simple string array, for example `["Row 1 column 1", "Row 1 column 2", "Row 1 column 3"]`.
+
+4) For each of these three row definitions, replace the entire string array with an object and set the object's `cells` property to the string array you had previously.
 
 For example: `["Row 1 column 1", "Row 1 column 2", "Row 1 column 3"]` becomes
 
-<pre class="file" data-target="clipboard">
+<pre class="file">
 {
   cells: ["Row 1 column 1", "Row 1 column 2", "Row 1 column 3"]
 }
 </pre>
 
-3) For each item in the cells array, replace the string value with an object whose title property is the cell data (a simple string in this case), just like you did in the previous two steps.
+Your rows definition should now look like:
```

Comment:
The rows definition in your table should now look like the table below: (or something im assuming thats a table below not an image)